modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition_manager.py

```diff
@@ -131,291 +130,27 @@ def wait_partitions(cls, partitions):
-        For your func to operate directly on the indices provided,
-        it must use `internal_indices` as a keyword argument.
-        """
-        return super(
-            PandasOnRayDataframePartitionManager, cls
-        ).apply_func_to_select_indices_along_full_axis(
-            axis, partitions, func, indices, keep_remaining
-        )
-
-    @classmethod
-    @progress_bar_wrapper
-    def apply_func_to_indices_both_axis(
-        cls,
-        partitions,
-        func,
-        row_partitions_list,
-        col_partitions_list,
-        item_to_distribute=no_default,
-        row_lengths=None,
-        col_widths=None,
-    ):
-        """
-        Apply a function along both axes.
-
-        Parameters
-        ----------
-        partitions : np.ndarray
-            The partitions to which the `func` will apply.
-        func : callable
-            The function to apply.
-        row_partitions_list : list
-            List of row partitions.
-        col_partitions_list : list
-            List of column partitions.
-        item_to_distribute : np.ndarray or scalar, default: no_default
-            The item to split up so it can be applied over both axes.
-        row_lengths : list of ints, optional
-            Lengths of partitions for every row. If not specified this information
-            is extracted from partitions itself.
-        col_widths : list of ints, optional
-            Widths of partitions for every column. If not specified this information
-            is extracted from partitions itself.
-
-        Returns
-        -------
-        np.ndarray
-            A NumPy array with partitions.
-
-        Notes
-        -----
-        For your func to operate directly on the indices provided,
-        it must use ``row_internal_indices`` and ``col_internal_indices`` as keyword
-        arguments.
-        """
-        return super(
-            PandasOnRayDataframePartitionManager, cls
-        ).apply_func_to_indices_both_axis(
-            partitions,
-            func,
-            row_partitions_list,
-            col_partitions_list,
-            item_to_distribute,
-            row_lengths,
-            col_widths,
-        )
-
-    @classmethod
-    @progress_bar_wrapper
-    def binary_operation(cls, left, func, right):
-        """
-        Apply a function that requires partitions of two ``PandasOnRayDataframe`` objects.
-
-        Parameters
-        ----------
-        left : np.ndarray
-            The partitions of left ``PandasOnRayDataframe``.
-        func : callable
-            The function to apply.
-        right : np.ndarray
-            The partitions of right ``PandasOnRayDataframe``.
-
-        Returns
-        -------
-        np.ndarray
-            A NumPy array with new partitions.
-        """
-        return super(PandasOnRayDataframePartitionManager, cls).binary_operation(
-            left, func, right
-        )
+def _make_wrapped_method(name: str):  # noqa: GL08
```

Comment:
We're saving so many lines, let's waste some on meaningful docstring for this function 🙃

Reply:
Docstring is added.